issue.go

```diff
@@ -78,6 +78,7 @@ type EditIssueOption struct {
 	Body      *string `json:"body"`
 	Assignee  *string `json:"assignee"`
 	Milestone *int64  `json:"milestone"`
+	Status    *string `json:"status"`
```

Comment:
@Unknwon should this be `json:"state"` instead? to reflect GitHub-API? https://developer.github.com/v3/issues/#edit-an-issue

In the gogs we call it "Status", which is why I named it thus.